data/json/npcs/Backgrounds/lost_partner_2.json

```diff
@@ -2,7 +2,11 @@
   {
     "id": "BGSS_LOST_PARTNER_2_STORY1",
     "type": "talk_topic",
-    "dynamic_line": "Just another <shitty> tale of love and loss.  Not something I like to tell.",
+    "dynamic_line": {
+      "npc_has_effect": "player_BGSS_SAIDNO",
+      "yes": "I said, I don't wanna talk about it.  How are you not understanding this?",
+      "no": "Just another <shitty> tale of love and loss.  Not something I like to tell."
```

Comment:
```suggestion
      "no": "Just another <shitty> tale of love and loss.  Not something I'd like to tell."
```

Reply:
That particular wording change I don't really agree with, but I did alter the structure a little to make my intent more clear.